djconnectwise/models.py

```diff
@@ -1552,9 +1570,25 @@ def update_cw(self, **kwargs):
             api_private_key=kwargs.get('api_private_key')
         )
 
-        return api_client.update_ticket(
```

Comment:
From the line 1559, the comments can be improved.

Reply:
Good call. I removed the references to only updating status and priority.

Reply:
I assume the version number which is now used can be also updated(2011.1 maybe?) or removed.

Reply:
Oh sorry I didn't realize you wanted that removed as well. 

I suppose there is not much value from saying WHEN we had to start sending updates to different API endpoints. I'll just change it to say:
>Send ticket status or priority and closed_flag updates to ConnectWise. Project tickets and issues need to be updated using the project API endpoint.

Is that ok with you?

Reply:
Looks great! Thanks~!